code/index.html

```diff
@@ -4,27 +4,83 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Dancing+Script:wght@500&family=Dongle:wght@300&family=Source+Sans+Pro:wght@300&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <!-- video or image as a header is cool :) -->
+    <header class="video-header">
+      <video autoplay loop muted poster="pic-bread.jpg">
+        <source src="video-bread.mp4" type="video/mp4" />
+        Your browser dosen't support the vido tag.
+      </video> 
+      <h1>Bagare Bengtssons bakbonanza!</h1>
+    </header>
+
+  <main>
+    <section class="grid-container">
+    <div class="grid-form">
+      <h2>Här anmäler du dig till årets roligaste bakkurs!</h2>
+      <form action="http://httpbin.org/anything" method="post">
+        
+      <label> Välj storlek på förkläde; <br>
+        <select name="size-apron">
+          <option value="s">Small</option> 
+          <option value="m">Medium</option>
+          <option value="l">Large</option>
+        </select> 
+      </label>
 
-    <!-- Signup form -->
+      <label class="labelmargin"> 
+        <br>
+        <input type="text" name="förOchEfternamn" placeholder=" För- och efternamn" required />
+      </label>
+      <label> 
+        <br>
+        <input type="email" name="email" placeholder=" E-mailadress" required />
+      </label>
+      <label> 
+        <br>
+        <input type="tel" name="telefonnummer" placeholder=" Telefonnummer" required />
+      </label>
+      <label class="labelmargin"> Något vi bör veta? 
+        <br>
+        <textarea></textarea>
+      </label>
+      <label class="labelmargin"> <a href="/link/"> Jag har läst och godkänner vilkoren</a> 
```

Comment:
if you are just making fake link and it does not actually lead anywhere you can use for example sign # like this:
<a href="#" id="theLink">My Link</a>